packages/unmock-node/src/persistence/snapshot.ts

```diff
@@ -39,31 +38,25 @@ expect.extend({
         CLEARED_SNAPSHOT_FILES.push(snapFile);
         fs.unlinkSync(snapFile);
       } else {
-        contents = require(snapFile);
+        contents = JSON.parse(fs.readFileSync(snapFile, "utf-8"));
       }
     }
     const numOfCalls = Object.keys(contents).length + 1;
     contents[`${this.currentTestName} ${numOfCalls}`] = obj;
-    const stringContents = Object.keys(contents)
-      .map(
-        (key: string) =>
-          "exports['" +
-          key +
-          "'] = " +
-          JSON.stringify(contents[key], undefined, 2) +
-          ";\n",
-      )
-      .join("\n");
+    const stringContents = JSON.stringify(contents, undefined, 2);
     fs.writeFileSync(snapFile, stringContents);
     return { pass: true };
   },
 });
 
-export const snapshot = (obj: any) => {
+export const snapshot = (obj: any, savePath?: string) => {
+  if (savePath === undefined) {
```

Comment:
This is a bit unexpected to me, to return immediately if an optional argument is not supplied? Usually optional arguments are auto-filled by the method if they're missing, if the user doesn't want anything to happen they don't call the method. But it's just a matter of style, no reason to change it!

Reply:
Good catch, it's a leftover :)
There's no point in passing an undefined `savePath` of course.